metron-deployment/README.md

```diff
@@ -15,178 +15,134 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 
-### Kerberos Without an MPack
-Using the MPack is preferred, but instructions for Kerberizing manually can be found at [Kerberos-manual-setup.md](Kerberos-manual-setup.md). These instructions are reference by the Ambari Kerberos install instructions and include commands for setting up a KDC.
 
-## TODO
-- Support Ubuntu deployments
+How do I deploy Metron on a single VM?
+--------------------------------------
+
+This will deploy Metron and all of its dependencies on a virtual machine running on your computer.  
+
+#### What is this good for?
+
+* If you are new to Metron and want to explore the functionality that it offers, this is good place to start.  
+
+* If you are a developer contributing to the Apache Metron project, this is also a great way to test your changes.  
+
+#### What is this **not** good for?
+
+* This VM is **not** intended for processing anything beyond the most basic, low volume work loads.
+
+* Additional services should **not** be installed along side Metron in this VM.
+
+* This VM should **not** be used to run a proof-of-concept for Apache Metron within your organization.
+
+Running Metron within the resource constraints of a single VM is incredibly challenging. Failing to respect this warning, will cause various services to fail mysteriously as the system runs into memory and processing limits. 
+
+#### How?
+
+To deploy Metron in a VM running on your computer, follow the instructions at [vagrant/full-dev-platform](vagrant/full-dev-platform)
+
+
+How do I build RPM packages?
+----------------------------
+
+This provides RPM packages that allow you to install Metron on an RPM-based operating system like CentOS.
+
+#### What is this good for?
+
+* If you want to manually install Apache Metron on an RPM-based system like CentOS, installation can be simplified by using these packages.  
+
+* If you want a guided installation process using Ambari on an RPM-based system, then these RPMs are a necessary prerequisite.
+
+#### What is this **not** good for?
+
+* If you want a complete, guided installation process, use Ambari rather than just these packages.  Installing Metron using **only** these RPMs still leaves a considerable amount of configuration necessary to get Metron running.  Installing with Ambari automates these additional steps.
+
+#### How?
+
+To build the RPM packages, follow the instructions at [packaging/docker/rpm-docker](packaging/docker/rpm-docker).
+
+
+How do I build DEB packages?
+-------------------------------
+
+This builds installable DEB packages that allow you to install Metron on an APT-based operating system like Ubuntu.
+
+#### What is this good for?
+
+* If you want to manually install Metron on a APT-based system like Ubuntu, installation can be simplified by using these packages.
+
+* If you want a guided installation process using Ambari on an APT-based system, then these RPMs are a necessary prerequisite.
```

Comment:
Minor correction.. I guess it should read DEBs instead of RPMs in this paragraph and the one below?